### DISCUSSION TITLE
Will Ungoogled-Chromium Be Able To Support AdBlocking After 2022?

### DISCUSSION THREAD

**Discussion by neurodiverseEsoteric** [Eloston/ungoogled-chromium#2192] [General]
No one seems to have already asked this in issues or discussions, which is weird because I would have thought news that Google would no longer allow ad-blocking or ad-blocking extensions as of 2023 and everyone on tumblr reblogging "switch to firefox" chants would have been something of pertinence for ungoogled-chromium's userbase and developers...

**Comment by jstkdng** [comment]
this issue is being discussed here https://github.com/ungoogled-software/ungoogled-chromium/issues/662

TLDR: Chromium is pushing back the date and if push comes to shove brave's changes will be ported to UC.

**Comment by neurodiverseEsoteric** [comment]
Ah okay thanks
